client/components/autocomplete_channels/index.coffee

```diff
@@ -1,5 +1,6 @@
 _ = require 'underscore'
 Backbone = require 'backbone'
+# TODO: Verify that this still works
 Modal = -> require('simple-modal') arguments...
```

Comment:
@kanaabe / @eessex - could one of you verify that this modal still works and get back to me? There was a weird issue that i ran into due to the library (`simple-modal`) using `cssify` internally. I had to exclude it in root `index.js`. But not sure if the css still applies to it

Reply:
I'm getting 
![image](https://user-images.githubusercontent.com/2236794/32575839-1b470498-c4a3-11e7-8d8a-3761b2c62ab3.png)
when I pull this down locally and open the Switch Channel modal. You can replicate it by hovering here:
![image](https://user-images.githubusercontent.com/2236794/32575864-300f6ce4-c4a3-11e7-8297-94b2c404e499.png)

Reply:
Maybe I'm missing something?